packages/darklang/stdlib/string.dark

```diff
@@ -206,8 +208,11 @@ module Darklang =
         let l = normalize ``to``
         let l = if f > l then f else l
 
-        (PACKAGE.Darklang.Stdlib.String.dropLast string (len - l))
-        |> PACKAGE.Darklang.Stdlib.String.dropFirst f
+        string
+        |> String.toList
```

Comment:
This isn't a good way to implement slice in dark (converting it to a list...)

I had this implementation, but dropLast and dropFirst use slice (infinite recursion)
```
(PACKAGE.Darklang.Stdlib.String.dropLast string (len - l))
  |> PACKAGE.Darklang.Stdlib.String.dropFirst f
 ```
 Any suggestions for implementing it without converting it into a list?

Reply:
I meant that PACKAGE.slice should call Builtin.slice, which does the efficient string slicing. PACKAGE.slice does the normalization, and Builtin.slice just does the bits we can only do in dotnet.

Reply:
(I just noticed that Builtin.slice is not efficient - but we can resolve that)